Include/pyerrors.h

```diff
@@ -65,6 +65,8 @@ PyAPI_FUNC(const char *) PyExceptionClass_Name(PyObject *);
 
 PyAPI_DATA(PyObject *) PyExc_BaseException;
 PyAPI_DATA(PyObject *) PyExc_Exception;
+PyAPI_DATA(PyObject *) PyExc_BaseExceptionGroup;
+PyAPI_DATA(PyObject *) PyExc_ExceptionGroup;
```

Comment:
I don't know what this is for, assuming it's needed.

Reply:
You mean the funky macro around the return type? It’s to make dynamic loaders publish the symbol on all platforms (but mostly Windows IIRC).